packages/core/src/configs/all.ts

```diff
@@ -0,0 +1,165 @@
+// THIS CODE WAS AUTOMATICALLY GENERATED
+// DO NOT EDIT THIS CODE BY HAND
+// SEE https://typescript-eslint.io/linting/configs
+//
+// For developers working in the typescript-eslint monorepo:
+// You can regenerate it using `yarn generate:configs`
```

Comment:
[Docs] This reminds me, `generate:configs` is only in `packages/eslint-plugin`...

Reply:
It's not any more - it's now a repo-tools script!